src/contracts/mixins/GPv2Signing.sol

```diff
@@ -22,6 +22,46 @@ library GPv2Signing {
     /// @dev Signing scheme used for recovery.
     enum Scheme {Eip712, EthSign, Eip1271}
 
+    /// @dev The EIP-712 domain type hash used for computing the domain
+    /// separator.
+    bytes32 private constant DOMAIN_TYPE_HASH =
+        keccak256(
+            "EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)"
+        );
+
+    /// @dev The EIP-712 domain name used for computing the domain separator.
+    bytes32 private constant DOMAIN_NAME = keccak256("Gnosis Protocol");
+
+    /// @dev The EIP-712 domain version used for computing the domain separator.
+    bytes32 private constant DOMAIN_VERSION = keccak256("v2");
+
+    /// @dev The domain separator used for signing orders that gets mixed in
+    /// making signatures for different domains incompatible. This domain
+    /// separator is computed following the EIP-712 standard and has replay
+    /// protection mixed in so that signed orders are only valid for specific
+    /// GPv2 contracts.
+    bytes32 public immutable domainSeparator;
+
+    constructor() {
```

Comment:
Do we know in which order the inherited constructors are called? Note that it necessarily matters here, just asking.

Reply:
You can specify this in the derived contract with something like `constructor() Foo(param) Bar(otherParam) { ... }`.